### PR TITLE
TL Localization - Chapter 6, section 5 (1,577 words)

### PR DESCRIPTION
**Pull request by heywens**

### CODE REVIEW

File: book/06-github/sections/5-scripting.asc [hunk 1]
```diff
@@ -96,34 +96,34 @@ end
...
-.Web hook debugging information.
-image::images/scripting-04-webhook-debug.png[Webhook debug]
+.Impormasyon sa pag-debug ng Web hook.
+image::images/scripting-04-webhook-debug.png[Pag-debug ng Webhook]
 
-The other great feature of this is that you can redeliver any of the payloads to test your service easily.
+Ang iba pang mahusay na tampok nito ay na maaari mong maibalik ang anumang mga kargamento upang madaling masubukan ang iyong serbisyo.
 
-For more information on how to write webhooks and all the different event types you can listen for, go to the GitHub Developer documentation at https://developer.github.com/webhooks/
+Para sa karagdagang impormosyon kung paano magsulat ng mga webhook at lahat ng iba't-ibang mga uri ng kaganapan na maaari mong pakinggan, pumunta sa dokumentasyon ng GitHub Developer sa https://developer.github.com/webhooks/
```
Comment (mind-blast) [inline book/06-github/sections/5-scripting.asc]: impormosyon  to impormasyon

File: book/06-github/sections/5-scripting.asc [hunk 2]
```diff
@@ -151,47 +151,47 @@ $ curl https://api.github.com/gitignore/templates/Java
   "name": "Java",
   "source": "*.class
 
-# Mobile Tools for Java (J2ME)
+# Kasangkapan sa Mobile para sa Java (J2ME)
 .mtj.tmp/
 
-# Package Files #
+# Mga Files ng Package #
```
Comment (mind-blast) [inline book/06-github/sections/5-scripting.asc]: Mga Files to Mga File

File: book/06-github/sections/5-scripting.asc [hunk 3]
```diff
@@ -151,47 +151,47 @@ $ curl https://api.github.com/gitignore/templates/Java
...
 
-However, if you want to do an action on the website such as comment on an Issue or Pull Request or if you want to view or interact with private content, you'll need to authenticate.
+Gayunpaman, kung nais mong gumawa ng isang aksyon sa website tulad ng komento sa isang Isyu o Kahilingan na Pull o kung nais mong tingnan o makipag-ugnayan sa pribadong nilalaman, kakailanganin mong magpatunay.
 
-There are several ways to authenticate.
-You can use basic authentication with just your username and password, but generally it's a better idea to use a personal access token.
-You can generate this from the ``Applications'' tab of your settings page.
+Mayroong maraming mga paraan upang magpatunay.
+Maaari mong gamitin ang pangunahing pagpapatunay gamit lamang ang iyong username at password, ngunit sa pangkalahatan mas magaling na ideya na gumamit ng isang access token na personal.
+Maaari mong mabuo ito mula sa tab na ``Mga Aplikasyon'' sa pahina ng iyong settings.
```
Comment (mind-blast) [inline book/06-github/sections/5-scripting.asc]: settings to mga setting

File: book/06-github/sections/5-scripting.asc [hunk 4]
```diff
@@ -240,63 +240,63 @@ require 'sinatra'
 require 'json'
 
 post '/payload' do
-  push = JSON.parse(request.body.read) # parse the JSON
+  push = JSON.parse(request.body.read) # i-parse ang JSON
   repo_name = push['repository']['full_name']
 
-  # look through each commit message
+  # tingan ang bawat mensahe ng commit
```
Comment (mind-blast) [inline book/06-github/sections/5-scripting.asc]: tingan to tingnan

**Comment by mind-blast** [comment]
Kindly check on my suggestions. Thank you.

**Comment by heywens** [comment]
Typos have been corrected. Thank you for pointing those out Sir @robin-ho.

**Pull request merged**